packages/pwa/app/components/action-card/index.jsx

```diff
@@ -45,12 +45,15 @@ const ActionCard = ({children, onEdit, onRemove, ...props}) => {
                 <Stack direction="row" spacing={4}>
                     {onEdit && (
                         <Button onClick={onEdit} variant="link" size="sm">
-                            <FormattedMessage defaultMessage="Edit" />
+                            <FormattedMessage defaultMessage="Edit" id="action_card.button.edit" />
```

Comment:
I think the UI type for this should be `action` and not `button`. A Button with variant=link and onClick:
- visually does not look like a button
- it does some action
- it's not a `link` either, because it doesn't really navigate you to another page

Reply:
yea It makes sense, I've changed it to `action`